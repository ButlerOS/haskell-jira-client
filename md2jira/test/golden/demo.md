---
title: Team backlog
---

Document usage:
- test

# Develop app  {#PROJ-42}

The goal of app is ...

```bash
$ command
output
```

## Implement X

The goal of this story is ...

- [ ] do x {.n}
- [ ] and y

what about:

- [ ] z task

## Implement Z

- [x] completed {.tc}
  > https://localhost

- [ ] todo
