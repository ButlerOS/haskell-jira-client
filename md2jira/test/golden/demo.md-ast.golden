Document
    { kind = FullBacklog
    , intro =
        [ Para
            [ Str "Document"
            , Space
            , Str "usage:"
            ]
        , BulletList
            [
                [ Plain
                    [ Str "test" ]
                ]
            ]
        ]
    , epics =
        [ Epic
            { mJira = Just "PROJ-42"
            , title = "Develop app"
            , description =
                [ Para
                    [ Str "The"
                    , Space
                    , Str "goal"
                    , Space
                    , Str "of"
                    , Space
                    , Str "app"
                    , Space
                    , Str "is"
                    , Space
                    , Str "..."
                    ]
                , CodeBlock
                    ( ""
                    , [ "bash" ]
                    , []
                    ) "$ command
                  output"
                ]
            , stories =
                [ Story
                    { mJira = Nothing
                    , title = "Implement X"
                    , assigned = []
                    , description =
                        [ Para
                            [ Str "The"
                            , Space
                            , Str "goal"
                            , Space
                            , Str "of"
                            , Space
                            , Str "this"
                            , Space
                            , Str "story"
                            , Space
                            , Str "is"
                            , Space
                            , Str "..."
                            ]
                        , BulletList
                            [
                                [ Plain
                                    [ Str "☐"
                                    , Space
                                    , Str "do"
                                    , Space
                                    , Str "x"
                                    , Span
                                        ( ""
                                        , [ "n" ]
                                        , []
                                        ) [ Space ]
                                    ]
                                ]
                            ,
                                [ Plain
                                    [ Str "☐"
                                    , Space
                                    , Str "and"
                                    , Space
                                    , Str "y"
                                    ]
                                ]
                            ]
                        , Para
                            [ Str "what"
                            , Space
                            , Str "about:"
                            ]
                        , BulletList
                            [
                                [ Plain
                                    [ Str "☐"
                                    , Space
                                    , Str "z"
                                    , Space
                                    , Str "task"
                                    ]
                                ]
                            ]
                        ]
                    , tasks =
                        [ Task
                            { status = Open
                            , title = "do x"
                            , assigned = [ "n" ]
                            , description =
                                [ Plain [] ]
                            }
                        , Task
                            { status = Open
                            , title = "and y"
                            , assigned = []
                            , description = []
                            }
                        , Task
                            { status = Open
                            , title = "z task"
                            , assigned = []
                            , description = []
                            }
                        ]
                    , mScore = Nothing
                    , updated = Nothing
                    , priority = Nothing
                    }
                , Story
                    { mJira = Nothing
                    , title = "Implement Z"
                    , assigned = []
                    , description =
                        [ BulletList
                            [
                                [ Para
                                    [ Str "☒"
                                    , Space
                                    , Str "completed"
                                    , Span
                                        ( ""
                                        , [ "tc" ]
                                        , []
                                        ) [ Space ]
                                    ]
                                , BlockQuote
                                    [ Para
                                        [ Str "https://localhost" ]
                                    ]
                                ]
                            ,
                                [ Para
                                    [ Str "☐"
                                    , Space
                                    , Str "todo"
                                    ]
                                ]
                            ]
                        ]
                    , tasks =
                        [ Task
                            { status = Closed
                            , title = "completed"
                            , assigned = [ "tc" ]
                            , description =
                                [ Plain []
                                , BlockQuote
                                    [ Para
                                        [ Str "https://localhost" ]
                                    ]
                                ]
                            }
                        , Task
                            { status = Open
                            , title = "todo"
                            , assigned = []
                            , description = []
                            }
                        ]
                    , mScore = Nothing
                    , updated = Nothing
                    , priority = Nothing
                    }
                ]
            }
        ]
    }